### DISCUSSION TITLE
When using fumadocs-typescript to generate tables, can I include other types imported by my TS file?

### DISCUSSION THREAD

**Discussion by arielweinberger** [fuma-nama/fumadocs#897] [Q&A]
Imagine I have this setup:

```ts
// lib.ts
export interface Something {
  prop: string;
} 
```

```ts
// class.ts
import { Something } from "./lib";

export class MyClass {
  something: Something;
}
```

When using fumadocs typescript to generate files following [this tutorial](https://fumadocs.vercel.app/docs/ui/typescript), I want to generate reference docs for MyClass but also want to include the spec for `Something` which is used inside of it.

Is there any way to automatically do this, or is the only way to explicitly have a `---type-table---` for the other interface?

**Comment by fuma-nama** [comment]
Hmm it's not implemented, PR welcome though
